### DISCUSSION TITLE
Windows 11 Skin

### DISCUSSION THREAD

**Discussion by PabloDaniel0** [bonzibudd/Fluent-Metro#166] [Ideas]
It is excellent the sikin, but it would be interesting if you could create one that has the design of the windows 11 start menu.
Would this be possible?

**Comment by bonzibudd** [comment]
I have an early version of a more Windows 11-oriented skin here: https://github.com/bonzibudd/Fluent-Refresh

It’s currently still very similar to Fluent-Metro, but it has some updated designs and sizings.

**Reply by PabloDaniel0** [reply]
Thanks, I will try it
